staging/src/k8s.io/client-go/discovery/discovery_client.go

```diff
@@ -377,17 +377,17 @@ func (d *DiscoveryClient) SwaggerSchema(version schema.GroupVersion) (*swagger.A
 
 // OpenAPISchema fetches the open api schema using a rest client and parses the json.
 // Warning: this is very expensive (~1.2s)
-func (d *DiscoveryClient) OpenAPISchema() (*spec.Swagger, error) {
-	data, err := d.restClient.Get().AbsPath("/swagger.json").Do().Raw()
+func (d *DiscoveryClient) OpenAPISchema() (*openapi_v2.Document, error) {
+	data, err := d.restClient.Get().AbsPath("/swagger-2.0.0.pb-v1").Do().Raw()
```

Comment:
Are you sure about the name.
I found `/swagger.pb` in https://docs.google.com/document/d/1EcnvLRqP9-5mqQ-kghfnB3GtB8eIrkVQjn1B16NRrr0/edit#heading=h.xgjl2srtytjt